.gitignore

```diff
@@ -257,7 +257,7 @@ cmake-build-debug
 # Below files are not deleted by "setup.py clean".
 
 # Visual Studio Code files
-.vscode
+/.vscode/settings.json
```

Comment:
`launch.json` and `c_cpp_properties.json` are now not ignored.

Reply:
I don't have those on my system; could you post their contents here (perhaps in a paste)? I don't think it's necessarily a bad thing to have those in the repo (thinking mostly of `launch.json`, I dunno what `c_cpp_properties.json` is), but I'm very curious to see their contents on your machine

Reply:
If it turns out that there are too many files that we want to ignore and we don't want to list them all, we could instead do something like this:
```gitignore
/.vscode/*
!/.vscode/extensions.json
!/.vscode/settings_recommended.json
```

Reply:
or if we do decide to ignore `launch.json` and `c_cpp_properties.json` but there aren't any others that we need to ignore, we could just add these two lines:
```gitignore
/.vscode/c_cpp_properties.json
/.vscode/launch.json
```